src/System.Text.Http.Parser/PipelineExtensions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.IO.Pipelines;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace System.Text.Http.Parser.Internal
+{
+    internal static class PipelineExtensions
```

Comment:
For now it it. The parser today depends on ReadableBuffer, which is part of pipelines. I will try to move the parser to ReadOnlyBytes in a subsequent PRs, but I need perf tests first.